train.py

```diff
@@ -42,15 +43,56 @@ def config():
   )
   learner = Learner.DEFAULT_CONFIG
   network = networks.DEFAULT_CONFIG
+  policy = policies.DEFAULT_CONFIG
   expt_dir = train_lib.get_experiment_directory()
 
+class TrainManager:
```

Comment:
This was moved to a separate file, I think you accidentally kept it when you merged.